bootcamp/contrib/tests/test_student15.py

```diff
@@ -0,0 +1,25 @@
+from bootcamp.contrib.student_15 import CaseConversion
+
+
+def test_AllUpper():
```

Comment:
Multi-word functions should be separated in snake case. See [PEP8](https://peps.python.org/pep-0008/#function-and-variable-names).

Reply:
Same for lines 12 and 20.